### PR TITLE
divide function

### PR DESCRIPTION
**Pull request by Jaiden16**

### CODE REVIEW

**Comment by KameronsCodeSpace** [comment]
I 2 I

**Review by mike923 — COMMENTED** [review]
If a = 0 the answer is just 0. It should only be if b is 0.

**Review by KameronsCodeSpace — APPROVED** [review]

**Review by mike923 — CHANGES_REQUESTED** [review]
if (b === 0)

File: divide.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const divide = (a,b) =>{
+    let result = a/b
+    if (a == 0 || b == 0){
```
Comment (KameronsCodeSpace) [inline divide.js]: (a === 0 || b === 0)

**Review by tbriany — APPROVED** [review]

**Review by KameronsCodeSpace — CHANGES_REQUESTED** [review]

**Pull request merged**